### PR TITLE
Update tests.js (cssom-view  & css-snappoints)

### PR DESCRIPTION
**Pull request by Zefling**
Add cssom-view and little change for css-snappoints

### CODE REVIEW

File: tests.js [hunk 1]
```diff
@@ -654,8 +661,8 @@ window.Specs = {
 			"scroll-snap-type": ["none", "mandatory", "proximity"],
 			"scroll-snap-type-x": ["none", "mandatory", "proximity"],
 			"scroll-snap-type-y": ["none", "mandatory", "proximity"],
-			"scroll-snap-points-x": ["none", "repeat(100%)", "repeat(120px), repeat(calc(3*25px))"],
-			"scroll-snap-points-y": ["none", "repeat(100%)", "repeat(120px), repeat(calc(3*25px))"],
+			"scroll-snap-points-x": ["none", "repeat(100%)", "repeat(120px)", "repeat(calc(3*25px))"],
+			"scroll-snap-points-y": ["none", "repeat(100%)", "repeat(120px)", "repeat(calc(3*25px))"],
```
Comment (LeaVerou) [inline tests.js]: I believe it was intentional to include multiple values on this property?

**Comment by LeaVerou** [comment]
Hey there! Thanks for contributing!
In the future, please send separate PRs for separate additions. For example, now I have to either merge both or none, and I’m not sure about the 2nd change (see inline comment).

File: tests.js (hunk 1 above)
Comment (Zefling) [inline tests.js]: It's a error, the draft no specify a multiple repeat()

**Comment by Zefling** [comment]
Sorry, I note this.

**Comment by LeaVerou** [comment]
Merged, thanks!

**Pull request merged**